app/uk/gov/hmrc/gform/submission/SubmissionService.scala

```diff
@@ -63,7 +63,7 @@ class SubmissionService(
   def submitForm(formIdData: FormIdData, customerId: String, submissionData: SubmissionData)(implicit
     hc: HeaderCarrier
   ): FOpt[Unit] =
-    // format: OFFLocalisedEmailTemplateId.scala
+    // format: OFF
```

Comment:
You didn't restore previous state, so this comment is not useful anymore, you can just as well removed it (and it's counterpart)